languages.json

```diff
@@ -36,6 +36,7 @@
     "kn": "ಭಾಷೆ-ಹೆಸರು",
     "rw": "Kinyarwanda",
     "ht": "Kreyòl",
+    "kk": "Қазақша",
```

Comment:
One note - this should be between 'kn' and 'rw',

Name-wise Kazakh, would be between Kannada and Kinyawanda.

Reply:
oups, thanks!